bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/api/AccountController.java

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.sunsynk.internal.api;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Properties;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.http.HttpMethod;
+import org.openhab.binding.sunsynk.internal.api.dto.APIdata;
+import org.openhab.binding.sunsynk.internal.api.dto.Client;
+import org.openhab.binding.sunsynk.internal.api.dto.Details;
+import org.openhab.binding.sunsynk.internal.api.dto.Inverter;
+import org.openhab.binding.sunsynk.internal.api.exception.SunSynkAuthenticateException;
+import org.openhab.binding.sunsynk.internal.api.exception.SunSynkInverterDiscoveryException;
+import org.openhab.binding.sunsynk.internal.api.exception.SunSynkTokenException;
+import org.openhab.core.io.net.http.HttpUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * The {@link AccountController} is the internal class for a Sunsynk Connect
+ * Account.
+ * 
+ * @author Lee Charlton - Initial contribution
+ */
+
+@NonNullByDefault
+public class AccountController {
+    private final Logger logger = LoggerFactory.getLogger(AccountController.class);
+    private final static int TIMEOUT_IN_MS = 4000;
+    private Client sunAccount = new Client();
```

Comment:
See SAT, change static <-> final keyword order.

```suggestion
    private static final int TIMEOUT_IN_MS = 4000;
    
    private final Logger logger = LoggerFactory.getLogger(AccountController.class);
    private Client sunAccount = new Client();
```

Reply:
done